src/xcode/ENA/ENA/Source/Models/Metadata/Test Result Metadata/TestResultMetadata.swift

```diff
@@ -19,14 +19,23 @@ struct TestResultMetadata: Codable {
 		
 		testResult = try container.decodeIfPresent(TestResult.self, forKey: .testResult)
 		hoursSinceTestRegistration = try container.decodeIfPresent(Int.self, forKey: .hoursSinceTestRegistration)
-		riskLevelAtTestRegistration = try container.decodeIfPresent(RiskLevel.self, forKey: .riskLevelAtTestRegistration)
-		daysSinceMostRecentDateAtRiskLevelAtTestRegistration = try container.decodeIfPresent(
+		enfRiskLevelAtTestRegistration = try container.decodeIfPresent(RiskLevel.self, forKey: .enfRiskLevelAtTestRegistration)
```

Comment:
Since you have renamed the key from "riskLevelAtTestRegistration" to "enfRiskLevelAtTestRegistration" is there some migration needed to preserve current data?

Reply:
Good point Mr. Database ;). It could be that we loose some data here. Because of that, I reverted the renaming.